### PR TITLE
Update sscs-venues.csv

### PR DESCRIPTION
**Pull request by MohammedHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9950

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/reference-data/sscs-venues.csv [hunk 1]
```diff
@@ -89,6 +89,7 @@
...
 201,"SC201","SSCS Cardiff","Havant Magistrates Court Building","The Courthouse","Elmleigh Road","","Havant","PO9 2AL","02 392 839 238",4503,"https://www.google.com/maps/place/Magistrates+Court/@50.8565386,-0.9854132,17z/data=!3m1!4b1!4m5!3m4!1s0x487444db5abdb8a9:0xd441b0067d2e7b28!8m2!3d50.8565352!4d-0.9832245","Yes","Havant",""
+203, "SC203", "SSCS Cardiff", "Southampton Crown Court", "The Courts of Justice", "London Road", "Southampton", "Hants", "SO15 2XQ", "", "4503", "https://www.google.com/maps/place/Southampton+Crown+Court/@50.9136832,-1.4033626,17.45z/data=!3m1!5s0x487476ac23b8b3e5:0x9e7862fbbec9109e!4m13!1m7!3m6!1s0x487476ac3b782539:0x5548c2a84b80bd0b!2sSouthampton+SO15+2XQ!3b1!8m2!3d50.9136969!4d-1.4029679!3m4!1s0x487476ac3c7dec37:0xc7e2f12290ad7393!8m2!3d50.9138088!4d-1.4029186", "Yes", "Southampton", ""
```
Comment (ss-evoco) [inline src/main/resources/reference-data/sscs-venues.csv:92]: "4503" should be 4503 (no quotes)

Comment (ss-evoco) [inline src/main/resources/reference-data/sscs-venues.csv:92]: Remove the space before each quote to match with the rest of the file.

Comment (ss-evoco) [inline src/main/resources/reference-data/sscs-venues.csv:92]: "4503" should be 4503 (no quotes)

**Review by ss-evoco — CHANGES_REQUESTED** [review]

File: src/main/resources/reference-data/sscs-venues.csv (hunk 1 above)
Comment (MohammedHMCTS) [reply src/main/resources/reference-data/sscs-venues.csv]: Good spot fixed

Comment (MohammedHMCTS) [reply src/main/resources/reference-data/sscs-venues.csv]: Removed the spaces

**Review by ss-evoco — APPROVED** [review]

**Pull request merged**